.github/workflows/tox.yml

```diff
@@ -39,6 +39,35 @@ jobs:
       - name: run pandas integration tests
         run: pip uninstall polars -y && pytest tests/integration/upstream_test.py::TestPandas
 
+  tox-modin:
```

Comment:
Modin's environment may not always be compatible with pandas and polars, so a separate job is required (perhaps it is unnecessary).